handsontable/src/helpers/dom/element.js

```diff
@@ -529,6 +529,33 @@ export function isVisible(element) {
   return true;
 }
 
+/**
+ * Returns true if the element has the height set to `0` or `0px` and overflow to `hidden` (height deliberately set to be 0).
+ *
+ * @param {HTMLElement} element The element to check.
+ * @returns {boolean} `true` if the element has height set to `0` or `0px` and overflow is set to `hidden`, `false` otherwise.
+ */
+export function hasZeroHeight(element) {
+  // If provided with a window-like object, return false.
+  if (element.document) {
+    return false;
+  }
+
+  const rootDocument = element.ownerDocument;
```

Comment:
Is that line needed? I tested it locally, and after removing all the tests, I passed.
```suggestion
  const rootDocument = element.ownerDocument;
```

Reply:
With the new usage, it shouldn't 👍